projects/objects/school_furniture/protos/Book.proto

```diff
@@ -10,375 +10,50 @@ PROTO Book [
-          32, -1, 35, 32, 33, -1, 16, 35, 33, -1,
-          16, 33, 22, -1, 24, 12, 19, -1, 24, 19,
-          30, -1, 25, 24, 30, -1, 25, 30, 31, -1,
-          13, 25, 31, -1, 13, 31, 20, -1, 26, 14,
-          17, -1, 26, 17, 28, -1, 27, 26, 28, -1,
-          27, 28, 29, -1, 15, 27, 29, -1, 15, 29,
-          18, -1, 39, 7, 14, -1, 39, 14, 26, -1,
-          38, 39, 26, -1, 38, 26, 27, -1, 3, 38,
-          15, -1, 38, 27, 15, -1, 36, 2, 12, -1,
-          36, 12, 24, -1, 37, 36, 24, -1, 37, 24,
-          25, -1, 6, 37, 13, -1, 37, 25, 13, -1,
-          37, 6, 39, -1, 6, 7, 39, -1, 36, 37,
-          38, -1, 37, 39, 38, -1, 2, 36, 3, -1,
-          36, 38, 3, -1, 8, 21, 10, -1, 21, 23,
-          10, -1, 9, 11, 16, -1, 9, 16, 22, -1,
-          11, 13, 16, -1, 13, 20, 16, -1, 0, 3,
-          15, -1, 0, 15, 8, -1, 9, 14, 4, -1,
-          14, 7, 4, -1, 5, 6, 13, -1, 5, 13,
-          11, -1, 10, 12, 1, -1, 12, 2, 1, -1,
-          4, 7, 6, -1, 4, 6, 5, -1, 1, 2,
-          3, -1, 3, 0, 1, -1, 9, 4, 11, -1,
-          4, 5, 11, -1, 0, 8, 1, -1, 8, 10,
-          1, -1, 10, 23, 19, -1, 8, 18, 21, -1,
-          9, 22, 17, -1
-        ]
-        texCoordIndex [
-          0, 1, 2, -1, 3, 4, 5, -1, 6, 7,
-          8, -1, 9, 10, 11, -1, 12, 13, 14, -1,
-          15, 16, 17, -1, 18, 19, 20, -1, 21, 22,
-          23, -1, 24, 25, 26, -1, 27, 28, 29, -1,
-          30, 31, 32, -1, 33, 34, 35, -1, 36, 37,
-          38, -1, 39, 40, 41, -1, 42, 43, 44, -1,
-          45, 46, 47, -1, 48, 49, 50, -1, 51, 52,
-          53, -1, 54, 55, 56, -1, 57, 58, 59, -1,
-          60, 61, 62, -1, 63, 64, 65, -1, 66, 67,
-          68, -1, 69, 70, 71, -1, 72, 73, 74, -1,
-          75, 76, 77, -1, 78, 79, 80, -1, 81, 82,
-          83, -1, 84, 85, 86, -1, 87, 88, 89, -1,
-          90, 91, 92, -1, 93, 94, 95, -1, 96, 97,
-          98, -1, 99, 100, 101, -1, 102, 103, 104, -1,
-          105, 106, 107, -1, 108, 109, 110, -1, 111, 112,
-          113, -1, 114, 115, 116, -1, 117, 118, 119, -1,
-          120, 121, 122, -1, 123, 124, 125, -1, 126, 127,
-          128, -1, 129, 130, 131, -1, 132, 133, 134, -1,
-          135, 136, 137, -1, 138, 139, 140, -1, 141, 142,
-          143, -1, 144, 145, 146, -1, 147, 148, 149, -1,
-          150, 151, 152, -1, 153, 154, 155, -1, 156, 157,
-          158, -1, 159, 160, 161, -1, 162, 163, 164, -1,
-          165, 166, 167, -1, 168, 169, 170, -1, 171, 172,
-          173, -1, 174, 175, 176, -1, 177, 178, 179, -1,
-          180, 181, 182, -1, 183, 184, 185, -1, 186, 187,
-          188, -1, 189, 190, 191, -1, 192, 193, 194, -1,
-          195, 196, 197, -1, 198, 199, 200, -1, 201, 202,
-          203, -1, 204, 205, 206, -1, 207, 208, 209, -1,
-          210, 211, 212, -1, 213, 214, 215, -1, 216, 217,
-          218, -1, 219, 220, 221, -1, 222, 223, 224, -1,
-          225, 226, 227, -1
-        ]
-        creaseAngle 1.1
       }
+    ]
+    name IS name
+    model "book"
+    boundingObject Box {
+      size 0.02 0.15 0.2
```

Comment:
The size also needs to be updated in the description at line 4:
https://github.com/cyberbotics/webots/blob/8829dcd2a0f07dc62b189bc9e74ce947b82c2eed/projects/objects/school_furniture/protos/Book.proto#L4